### PR TITLE
drunktalk SO

### PR DESCRIPTION
**Pull request by VernaSaurus**
not actually connected to anything, but i did leave some things that should probably(?) help with implementing it

oh also i added inhibitors

### CODE REVIEW

File: UnityProject/Assets/Resources/ScriptableObjects/Speech/CustomMods/SlurredMod.cs [hunk 1]
```diff
@@ -1,10 +1,33 @@
...
 using UnityEngine;
+using AdminTools;
 
 [CreateAssetMenu(fileName = "CustomSpeechModifierCode", menuName = "ScriptableObjects/SpeechModifiers/SlurredSpeech")]
 public class SlurredMod : CustomSpeechModifier
 {
-	// i also need to randomly insert burps and hiccups
+	public int drunkSpeechTime;
+	public Mind playerMind;
+	private void OnEnable()
```
Comment (Bod9001) [inline UnityProject/Assets/Resources/ScriptableObjects/Speech/CustomMods/SlurredMod.cs]: make sure to clear out empty functions

Comment (VernaSaurus) [reply UnityProject/Assets/Resources/ScriptableObjects/Speech/CustomMods/SlurredMod.cs]: they aren't empty????

Comment (VernaSaurus) [reply UnityProject/Assets/Resources/ScriptableObjects/Speech/CustomMods/SlurredMod.cs]: oh whoops now i see what you mean

**Comment by VernaSaurus** [comment]
it done and can be merge now

**Pull request merged**